tests/sentry/runner/commands/test_configoptions.py

```diff
@@ -96,22 +89,22 @@ def assert_output(rv):
             # independently.
             output_before_log = "\n".join(
                 [
-                    SET_MSG % ("int_option", 40),
-                    UPDATE_MSG % ("str_option", "old value", "new value"),
-                    SET_MSG % ("map_option", {"a": 1, "b": 2}),
-                    SET_MSG % ("list_option", [1, 2]),
-                    DRIFT_MSG % "drifted_option",
+                    ConsolePresenter.SET_MSG % ("int_option", 40),
+                    ConsolePresenter.UPDATE_MSG % ("str_option", "old value", "new value"),
+                    ConsolePresenter.SET_MSG % ("map_option", {"a": 1, "b": 2}),
+                    ConsolePresenter.SET_MSG % ("list_option", [1, 2]),
+                    ConsolePresenter.DRIFT_MSG % "drifted_option",
                 ]
             )
 
             output_after_log = "\n".join(
                 [
-                    DB_VALUE % "drifted_option",
+                    ConsolePresenter.DB_VALUE % "drifted_option",
                     "- 1",
                     "- 2",
                     "- 3",
                     "",
-                    CHANNEL_UPDATE_MSG % "change_channel_option",
+                    ConsolePresenter.CHANNEL_UPDATE_MSG % "change_channel_option",
                 ]
             )
             assert output_before_log in rv.output
```

Comment:
Not sure why, but since moving the logging into the presenter this is breaking. The ConsolePresenter no longer pollutes the output with output_before_log.

Reply:
jk the output isnt lost, its just reformatted. I'll update the test cases to reflect this.